packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -120,11 +121,79 @@ const publicEventSelect = Prisma.validator<Prisma.EventTypeSelect>()({
       timeZone: true,
     },
   },
+  instantMeetingSchedule: {
+    select: {
+      id: true,
+      timeZone: true,
+    },
+  },
+
   hidden: true,
   assignAllTeamMembers: true,
   rescheduleWithSameRoundRobinHost: true,
 });
 
+export async function isCurrentlyAvailable({
```

Comment:
This function is just to check if current time is between the available time. It doesn't check if users are already booked, calendar blocked, OOO. 

As this is just for instant booking event where user can book for future slots if no one is available currently